Add info about CLI and SDK (#503)
livingdocsIO/documentation: content/reference-docs/cli/_index.md

{{< github "livingdocsIO/livingdocs-cli" "Livingdocs command line utility" >}}

## Documentation

> The CLI is used to modify setting in a live Livingdocs project, without the need to touch its code. 

Reviewer: ```suggestion
The `livingdocs-cli` is used to update a Livingdocs Project Config. 
```
